routes/clients.js

```diff
@@ -195,6 +195,43 @@ router.delete('/:idClient/products/:idProduct/favorites', (req, res) => {
             res.sendStatus(200)
         }
     })
+});
+
+/////////////////////////////////////////////////////// Favorites shops //////////////////////////////////////////////////////////
+
+//GET all the favorite shops of one client by it's ID
+router.get('/:idClient/favorites/shops', (req, res) => {
```

Comment:
Pas obligé d'ajouté dans l'uri "shops" (car on mets toujours en favoris un produit) sauf erreur de ma part
```suggestion
router.get('/:idClient/favorites', (req, res) => {
```